client/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "ion-extension-client",
-	"version": "0.0.1",
+	"version": "0.2.0",
```

Comment:
Just checking here, you did not manually modify the `package-lock.json`, right? It is automatically managed, see https://docs.npmjs.com/configuring-npm/package-lock-json.html

Reply:
I did manually modify it.

Reply:
I will remove this change.

Reply:
Read the link I pasted in my comment. Letting npm manage package-lock.json and then adding it to the repo is what we should do here.